src/send/EnterAmount.tsx

```diff
@@ -71,10 +72,13 @@ interface Props {
   prepareTransactionError?: Error
   tokenSelectionDisabled?: boolean
   onPressProceed(args: ProceedArgs): void
+  onPressInfo?(): void
```

Comment:
is `onPressInfo` needed in EnterAmount? Seems like it just gets it as a prop and passes in back to proceed component, which can all be done within EarnEnterAmount

Reply:
no, I'll update it to just be within the proceed component

Reply:
actually, I think to do this I would have to have the EarnProceed function inside EarnEnterAmount, and this causes issues with the token amount state value and also rendering APY.

So I think EarnProceed (which needs access to onPressInfo) needs to be outside of EarnEnterAmount so I'm not sure then how it would have access to onPressInfo. @satish-ravi is there some way to access it from outside the function it is declared without passing as a prop?